translations/de.json

```diff
@@ -108,6 +108,7 @@
   "show/hide": "Anzeigen/Verbergen",
   "skiing": "Skifahren",
   "snowboarding": "Snowboarden",
+  "golfing": "Golfen",
   "swimming": "schwimmen",
```

Comment:
I assume the correct German for is capitalized "Schwimmen"; would you mind adding that extra fix to your PR?

Reply:
fixed it